pyproject.toml

```diff
@@ -14,10 +14,10 @@ maintainers = [{ name = "PyMC Labs", email = "info@pymc-labs.io" }]
 dependencies = [
     "arviz>=0.13.0",
     "matplotlib>=3.5.1",
-    "numpy>=1.17",
+    "numpy>=1.17,<1.25",
```

Comment:
We shouldn't have to pin numpy and PyTensor does it already

Reply:
Done!